docs/core/diagnostics/runtime-type-events.md

```diff
@@ -0,0 +1,36 @@
+# Runtime Type Events
```

Comment:
All .md files need to have a markdown section at the top with ms.date, title, and description. Also, you can put `ms.topic: reference`.